sql/moz-fx-data-shared-prod/static/standardized_country_names/description.txt

```diff
@@ -0,0 +1 @@
+Mappings from country codes or non-standard names to a standardized country name.
```

Comment:
We should make it clear that this is GA-specific, considering we have some odd entries in here like "worldwide" and "(not set)".  I would recommend changing the name of the table to `ga_standardized_country_names_v1` and mentioning Google Analytics specifically in this description.